imap_processing/cli.py

```diff
@@ -154,7 +150,7 @@ def _validate_args(args):
     if args.instrument not in imap_data_access.VALID_INSTRUMENTS:
         raise ValueError(
             f"{args.instrument} is not in the supported instrument list: "
-            f"{imap_processing.INSTRUMENTS}"
+            f"{imap_data_access.VALID_INSTRUMENTS}"
         )
     if args.data_level not in imap_processing.PROCESSING_LEVELS[args.instrument]:
```

Comment:
should we also use valid data level from `imap-data-access`?

Reply:
I looked at this, and the VALID_PROCESSING_LEVELS is generic over in that repository and not "per instrument" like we are checking here. So I held off on that for now.

I agree though, we should look at making that update, but I think we should push it off to a follow-up PR.

Reply:
Yeah, that sounds good!